exporter/kafkaexporter/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter
+
+go 1.16
+
+require (
+	github.com/Shopify/sarama v1.29.1
+	github.com/gogo/protobuf v1.3.2
+	github.com/jaegertracing/jaeger v1.25.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-00010101000000-000000000000
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.0.0-20210821065208-3703e0d1dcaa
```

Comment:
Should this still be referencing `00010101...` with a `replace` directive?

Reply:
Using `00010101...` would be fine for jaeger, but if we use this for opencensus, an error like `github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000` will occur.

Reply:
You need that same replace statement in the root go.mod

Reply:
Right, makes sense. I have made the changes.